CHANGELOG.md

```diff
@@ -1,6 +1,15 @@
 
 # Changelog
 
+## 2.5.3
+* Basic PPS UI by @bbonenfant in https://github.com/pachyderm/pachyderm/pull/8557
```

Comment:
something cleaner:

Put helm release info into debug dumps - #8620
Add some miscellaneous debugging tools to "pachctl misc" - #8619
debug: avoid putting helm release info in profile/binary dumps - #8662
error handling for retrieveCallInfo - #8666
Fix auth activation from the environment - #8683
cron: log ticks at INFO level; wrap errors in related code for easier debugging - #8690
Fix project panic in pps.fixPipelineACL - #8688
Avoid setting LOG_LEVEL in the worker - #8694
Update pachctl stop job to use job's actual project instead of default - #8695